cmd/dashboard.go

```diff
@@ -78,6 +105,21 @@ func openInBrowser(url string) error {
 	return cmd.Start()
 }
 
+func serverResourceForGroupVersionKind(disco discovery.DiscoveryInterface, gvk schema.GroupVersionKind) (*metav1.APIResource, error) {
```

Comment:
Oh we should ask Gus to open this function on kubecfg later then we won't have to copy it here. But for now I am fine with leaving it here for unblocking the PR

Reply:
yeah that would be useful